griptape/tools/email_client/tool.py

```diff
@@ -100,25 +100,30 @@ def retrieve(self, params: dict) -> ListArtifact | ErrorArtifact:
     )
     def send(self, params: dict) -> InfoArtifact | ErrorArtifact:
         values = params["values"]
+        if self.smtp_user is None:
+            return ErrorArtifact("smtp_user is required")
```

Comment:
We should update the attribute helper comment at the top of this file to specify when these Optional parameters are actually required.